src/context/DashboardWelcome.tsx

```diff
@@ -0,0 +1,42 @@
+import { Dispatch, SetStateAction, createContext, useContext } from 'react';
+import { useLocalStorage } from 'react-use';
+import { BaseComponentProps } from 'types';
+import { LocalStorageKeys } from 'utils/localStorage-utils';
+
+interface DashboardWelcomeState {
+    welcomeShown: boolean | undefined;
+    setWelcomeShown: Dispatch<SetStateAction<boolean | undefined>>;
+}
+
+const DashboardWelcomeContext = createContext<DashboardWelcomeState | null>(
+    null
+);
+
+const DashboardWelcomeProvider = ({ children }: BaseComponentProps) => {
+    const [welcomeShown, setWelcomeShown] = useLocalStorage(
+        LocalStorageKeys.DASHBOARD_WELCOME,
+        false
+    );
```

Comment:
Why are we making a provider for this?

Reply:
I was unsure whether the contents of the provider would be extended. Originally, I anticipated a partial redesign of the welcome section and didn't want to strip it until I saw what was replacing the data trends chart/how Jonathan was planning to extend the dashboard.